src/core/symbology-ng/qgsrenderer.h

```diff
@@ -263,7 +263,7 @@ class CORE_EXPORT QgsFeatureRenderer
     Q_DECL_DEPRECATED virtual QDomElement writeSld( QDomDocument& doc, const QgsVectorLayer &layer ) const;
     //! create the SLD UserStyle element following the SLD v1.1 specs with the given name
     //! @note added in 2.8
-    virtual QDomElement writeSld( QDomDocument& doc, const QString& styleName ) const;
+    virtual QDomElement writeSld( QDomDocument& doc, const QString& styleName, QgsStringMap props = QgsStringMap() ) const;
```

Comment:
It changes the API because it's virtual, so any override of it will no longer match in signature.